### PR TITLE
Correct Samsung Internet support for `appearance`

### PR DESCRIPTION
**Pull request by lukewarlow**
#### Summary

Corrected the Samsung Internet support data for the `appearance` property.

#### Test results and supporting details

Just copied across blink support data.

### CODE REVIEW

File: css/properties/appearance.json [hunk 1]
```diff
@@ -347,10 +352,15 @@
                 "version_added": "1",
                 "partial_implementation": true
               },
-              "samsunginternet_android": {
-                "version_added": "1.0",
-                "partial_implementation": true
-              },
+              "samsunginternet_android": [
+                {
+                  "version_added": "13.2"
```
Comment (queengooborg) [inline css/properties/appearance.json:357]:
```suggestion
                  "version_added": "13.0"
```

File: css/properties/appearance.json [hunk 2]
```diff
@@ -506,10 +516,15 @@
...
```
Comment (queengooborg) [inline css/properties/appearance.json:521]:
```suggestion
                  "version_added": "13.0"
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Good catch, thank you!  I'm assuming the version numbers were based upon mirroring, could we change the version numbers from "13.2" to "13.0" (which matches Chrome 83)?  If you've done testing or have found documentation that indicates it was actually added in 13.2, feel free to disregard my change request!

File: css/properties/appearance.json (hunk 2 above)
Comment (lukewarlow) [reply css/properties/appearance.json]: Good catch I just assumed 13.0 was on an Earlier chromium version. Corrected that now

**Review by queengooborg — APPROVED** [review]
Thank you for the quick fix, this is looking good to me!  Aside from 1.5, a Chrome engine bump has happened in every major Samsung Internet release, though some features were introduced in the minor releases.

**Pull request merged**